tests/unit/test_stale_symbol_surfaces.py

```
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+import importlib.util
+import sys
+from pathlib import Path
+
+
+def _load_check_module():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_load_check_module`

(ANN202)